From 01161473ed33ee682f340e6f12edcba5334475e0 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 30 Apr 2012 06:09:30 -0400
Subject: [PATCH] afp: Use g_clear_object properly

g_clear_object expects the address of a pointer. A recent
change in its definition makes gcc complain about this.
(cherry picked from commit 7fe5deeae25f2396027f667d8960d2ec84c64716)
---
 daemon/gvfsafpvolume.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/daemon/gvfsafpvolume.c b/daemon/gvfsafpvolume.c
index 3fd33f2..5207459 100644
--- a/daemon/gvfsafpvolume.c
+++ b/daemon/gvfsafpvolume.c
@@ -1082,7 +1082,7 @@ create_directory_get_filedir_parms_cb (GObject *source_object, GAsyncResult *res
   return;
 
 error:
-  g_clear_object (info);
+  g_clear_object (&info);
   g_simple_async_result_take_error (simple, err);
   g_simple_async_result_complete (simple);
   g_object_unref (simple);
@@ -3033,4 +3033,4 @@ g_vfs_afp_volume_read_from_fork_finish (GVfsAfpVolume  *volume,
     *bytes_read = g_simple_async_result_get_op_res_gssize (simple);
   
   return TRUE;
-}
\ No newline at end of file
+}
-- 
1.7.8.6

